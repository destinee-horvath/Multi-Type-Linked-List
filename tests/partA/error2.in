NEW 3
three 
2.0
1
REMOVE 3
REMOVE 2