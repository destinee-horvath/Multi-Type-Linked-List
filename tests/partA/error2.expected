INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE